src/backend/marsha/core/api/video.py

```diff
@@ -99,6 +102,14 @@ def get_serializer_context(self):
 
         return context
 
+    def perform_update(self, serializer):
+        super().perform_update(serializer)
+        channel_layer = get_channel_layer()
+        async_to_sync(channel_layer.group_send)(
+            VIDEO_ROOM_NAME.format(video_id=serializer.data["id"]),
+            {"type": "video_updated"},
+        )
```

Comment:
I was unsuccessfully looking for a way to test this without mocking.
Do you think we should ?

Reply:
I think we should, as your last commit modifications on `video_update` are not reflected here.
```suggestion
    def perform_update(self, serializer):
        super().perform_update(serializer)
        channel_layer = get_channel_layer()
        async_to_sync(channel_layer.group_send)(
            VIDEO_ROOM_NAME.format(video_id=serializer.data["id"]),
            {"type": "video_updated", "video": serializer.data},
        )
```

Reply:
Yes but I didn't push the fixup related to this part yet